pycsw/ogc/csw/csw2.py

```diff
@@ -33,10 +33,15 @@
 import os
 import sys
 import cgi
-from urllib2 import quote, unquote
-import urlparse
-from cStringIO import StringIO
-from ConfigParser import SafeConfigParser
+try:
+    from urllib2 import quote, unquote
+except ImportError:
+    from urllib.parse import quote, unquote
```

Comment:
Since there's a `six` import just below here, you can just import this from `six.moves` without the `try/except`.